tools/toascii/main.cpp

```diff
@@ -57,8 +56,10 @@ static void PrintUsage(const char* exe_name)
 #endif
 }
 
-static std::string GetOutputFileName(const gfxrecon::util::ArgumentParser& arg_parser,
-                                     const std::string&                    input_filename)
+namespace
```

Comment:
Why did you add this namespace around this function?

Reply:
TIL about anonymous namespace in a compilation unit!  It will be the first time it's used in any of our tools, but I suppose there's no harm!

Reply:
This was leftover from when I had my path split function in here. I can follow-up with a tiny PR to put things back to `static` or widen the anon namespace up to cover the constants and other static linkage function depending on your preference.
TIL that there was a backlash against the deprecation of this use of static in favour of anon namespaces and that the current consensus is _use whichever your prefer as we'll never be taking this use of static out of the standard due to C compatibility and too much old C++ using it_.